Client/Components/LoginFormComponent.razor

```diff
@@ -8,16 +8,16 @@
 @inject IToastService _toastService
 
 <EditForm Model="@_loginDto" OnValidSubmit="@HandleValidSubmit" autocomplete="off">
-    <DataAnnotationsValidator/>
+    <DataAnnotationsValidator />
     <div class="form-group">
-        <label for="email-input" class="d-block">Email Address</label>
-        <InputText id="email-input" @bind-Value="_loginDto.Email"/>
-        <ValidationMessage For="() => _loginDto.Email"/>
+        <label for="email-input" class="d-block">Nickname or Email</label>
+        <InputText id="email-input" @bind-Value="_loginDto.NicknameOrEmail" />
+        <ValidationMessage For="() => _loginDto.NicknameOrEmail" />
     </div>
     <div class="form-group">
         <label for="password-input" class="d-block">Password</label>
```

Comment:
missing type="password"

Reply:
Whole component will be changed in future project task, this pull request refers only to login system